lib/srv/db/access_test.go

```diff
@@ -839,6 +840,9 @@ func setupTestContext(ctx context.Context, t *testing.T, withDatabases ...withDa
 			testCtx.fakeRemoteSite,
 		},
 	}
+	// Empty config means no limit.
+	connLimiter, err := limiter.NewConnectionsLimiter(limiter.Config{})
```

Comment:
@smallinsky Yes, #7893 says "rate limiting", but this issue mention also application access which is not implemented as a part of this PR. I followed the pattern implemented in other places where only connection limiting is enforced, example:
desktop access: https://github.com/gravitational/teleport/blob/ce03d2cdeab117a69ccd51c90d21adc66fcaf04e/lib/service/desktop.go#L190
ssh: 
https://github.com/gravitational/teleport/blob/v7.0.0/lib/sshutils/server.go#L421

I also think that connection limiting is more useful for DB access as DB connections are long living. Although I can see some cases where rate limiting may be beneficial. Adding it probably won't hurt, but then we will be inconsistent (as mentioned above desktop access only limits connections too).

@r0mant Any thoughts?

Reply:
If this is a matter of drop-in replacement of "limiter.NewConnectionsLimiter" with "limiter.NewLimiter", I'd say let's go for it. The fact that desktop access doesn't use it may just be an oversight.